packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -14,6 +14,15 @@ import {
 import { HIGH_SEVERITY, MEDIUM_SEVERITY, LOW_SEVERITY } from './Constants';
 import ComplianceRemediationButton from './ComplianceRemediationButton';
 
+export const columns = [
+    { title: 'Rule', transforms: [ sortable ] },
+    { title: 'Policy', transforms: [ sortable ] },
+    { title: 'Severity', transforms: [ sortable ] },
+    { title: 'Passed', transforms: [ sortable ] },
+    { title: <React.Fragment><Ansible size='lg'/> Ansible</React.Fragment>,
```

Comment:
The icon is too big compared to the text. Try medium, or maybe the default would inherit font-size.

![Screenshot_2020-10-02 Compliance](https://user-images.githubusercontent.com/7695766/94916001-1fd7f780-04ae-11eb-8f73-b98265b8e056.png)

Reply:
Changed to 'sm' that should fit better.

Reply:
I guess the medium would fit better, but I'll leave it to your discretion.